Tutorials/i2c.md

```diff
@@ -0,0 +1,110 @@
+# I2C
+
+I2C is a simple-to-use communication protocol which can have more than one master, with the upper bus speed defined, and only two wires with pull-up resistors are needed to connect almost _unlimited_ number of I2C devices.
+
+Each slave device has a unique address. Transfer from and to master device is serial and it is split into 8-bit packets. All these simple requirements make it very simple to implement I2C interface even with cheap microcontrollers that have no special I2C hardware controller. You only need 2 free I/O pins and few simple I2C routines to send and receive commands.
+
+Let us get the accelerometer values using the I2C protocol. We will be using the [The Tessel Accelerometer](https://www.seeedstudio.com/Tessel-Accelerometer-Module-p-2223.html)
+
+[Refer to the datasheet for the accelerometer module to follow along](http://www.nxp.com/docs/en/data-sheet/MMA8452Q.pdf)
+
+![Fritzing Diagram](http://i.imgur.com/zK4U4S3.png)
```

Comment:
A potentially more useful Fritzing diagram (in addition to this one): since we're just using the I2C pins, what if you draw a fritzing diagram where just the pins you actually need are connected by wires between module and Tessel? And possibly label red/black wires for power vs SCL/SDA in whatever color you make them